pkg/libvirttools/virtualization.c

```diff
@@ -20,16 +20,64 @@ limitations under the License.
 #include "virtualization.h"
```

Comment:
Well, IMO the criteria here should be "is there any logic added which is more convinient to be implemented in c or feasable only using c", because you know it's still "Go project" so it's likely to have calls from using cgo. However, it's not critical at all. :)